ice_fishing_abm_1/ice_fishing_abm_gp/tests/test_movement_destination_subroutine.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import numpy as np
+from pytest import fixture
+from ice_fishing_abm_1.ice_fishing_abm_gp.movement_destination_subroutine import ExplorationStrategy, GPExplorationStrategy
+
+@fixture
+def exploration_strategy():
+    return ExplorationStrategy()
+
+@fixture
+def gp_exploration_strategy():
+    return GPExplorationStrategy()
+
+def test_initialization_exploration_strategy(exploration_strategy):
```

Comment:
I don't think we need to test the default parameters. The initialization is already tested in the fixture

Reply:
accepted